Dockerfile

```diff
@@ -25,9 +25,9 @@ RUN pip install --upgrade --no-cache-dir -r requirements/pip.txt && \
 COPY . /app
 RUN NODE_ENV=production yarn build && \
     DJANGO_CONFIGURATION=Build ./manage.py collectstatic --no-input && \
-    mkdir -p media && chown app:app media && \
-    mkdir -p /test_artifacts && \
-    chmod 777 /test_artifacts
+    mkdir -p media && chown app:app media
```

Comment:
Media is the place where the webapp can write files, especially uploaded files. This doesn't matter in our prod builds, since they go to S3, but it is important for other uses of this docker file.